pkg/venafi/tpp/connector_test.go

```diff
@@ -54,7 +55,7 @@ func init() {
 
 	resp, err := tpp.GetRefreshToken(&endpoint.Authentication{
 		User: ctx.TPPuser, Password: ctx.TPPPassword,
-		Scope: "certificate:approve,delete,discover,manage,revoke;"})
+		Scope: "certificate:approve,delete,discover,manage,revoke;configuration"})
```

Comment:
@brantpeery which method(s) are you using that require the `configuration` scope?  Most of the metadata methods are scopeless and that decision was recently affirmed by the TPP architects.

Also, not introduced here but something we should fix, there is no `approve` restriction defined for the `certificate` scope.

Reply:
Looks like it is `GET SystemStatus/Version`.  I'll check that with the TPP architect, too.

Reply:
@tr1ck3r @BrandonRoehl I checked in documentation, SystemStatus/Version really needs configuration scope. The problem here that if we merge this pull request any request with custom fields will need this scope.

Reply:
Maybe change RequestSystemVersion to the additional request flag, something like pre192-cutom-field? It is not so convenient, but don't need configuration scope.